frontends/web/src/routes/account/send/send.tsx

```diff
@@ -379,11 +379,12 @@ class Send extends Component<Props, State> {
         } else if (target.id === 'amount') {
             this.convertToFiat(value);
         }
-        this.setState(prevState => ({
+        this.setState((prevState) => ({
```

Comment:
Some times I see brackets like here in `(prevState)` but some other times I don't, like in a previous commit in `{element => this.elem = element}`? What's the difference? Shouldn't it always be in the same way for better style consistency?